java/code/src/com/suse/manager/webui/controllers/contentmanagement/response/ProjectSourceResponse.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.suse.manager.webui.controllers.contentmanagement.response;
+
+/**
+ * JSON response wrapper for the sources of a content project.
+ */
+// After adding more project source types it might be handy to break this bean into 2
+public class ProjectSourceResponse {
+
+    private String id;
```

Comment:
Any reason why the Id is a `String` instead of a `Long` or `Int`?

Reply:
No big reason, in the end, everything is going to be a string when you convert it to JSON, so having it like that you have more control how you convert the id to a String, but I can change it to a Long.